### ISSUE TITLE
Oversized /var/log/syslog should be cleaned op

### ISSUE THREAD

**Issue by hanaral** [elementary/os#363]
## Prerequisites
- [ ] I have searched open and closed issues for duplicates.

## Describe the bug

If applicable, add screenshots or screen recordings to help explain your problem.

## Platform Information

- [ ] I'm using the latest version from git that I've manually compiled
- [ ] I'm using the latest released stable version

**Comment by fd6130** [comment]
Yeah this log file size issue very annoying. Whenever my wifi connection got issue and during reconnecting it will keep write log and cause the log size become over 50GB...
